packages/react-native-builder-bob/src/index.ts

```diff
@@ -418,7 +418,7 @@ yargs
             root,
             source: path.resolve(root, source as string),
             output: path.resolve(root, output as string, 'commonjs'),
-            options: targetOptions,
+            options: { ...targetOptions, exclude: options.exclude },
```

Comment:
We are use the pattern of passing top-level options in the argument object (like `source`, `output`) and target specific options in the `options` field. So lets do the same for `exclude`.